dbt_platform_helper/commands/pipeline.py

```diff
@@ -92,7 +92,12 @@ def generate():
 def _clean_pipeline_config(pipelines_dir):
     if pipelines_dir.exists():
         click.echo("Deleting copilot/pipelines directory.")
-        rmtree(pipelines_dir)
+        try:
+            rmtree(pipelines_dir)
+            click.echo("Deleted!")
+        except Exception as e:
+            click.echo(f"Failed to delete {pipelines_dir}: {e}")
```

Comment:
I'm curious as to how you got here?

Reply:
Sorry just some crap from when I was testing that intermittent actions test failure a few days ago